src/com/mendix/widget/carousel/components/__tests__/CarouselItem.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { ShallowWrapper, shallow } from "enzyme";
+import { createElement } from "react";
+
+import { CarouselItem, CarouselItemProps } from "../CarouselItem";
+
+describe("CarouselItem", () => {
+    const url = "data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7";
+    let output: ShallowWrapper<CarouselItemProps, any>;
+    describe("active", () => {
+        beforeEach(() => output = shallow(createElement(CarouselItem, { url, active: true })));
+
+        it("should be a div with one img child", () => {
```

Comment:
This is a general tests as it also applies to inactive items.

There is no need to make two describe blocks.

The tests I expect are:
- it renders an image (verify url is set in an img tag)
- it renders the structure correctly
- it renders the item css class 
- it renders the active css class when active